samples/quickstart/FastqToUbamSingleSample.wdl

```diff
@@ -60,7 +60,8 @@ task FastqToUbam {
     runtime {
         docker: docker_image
         memory: "32 GB"
-        cpu: "8"
+        cpu: "16"
+        disk: "500 GB"
```

Comment:
Consider changing this to depend on file size with buffer instead of hard coding to 500GB. @olesya13

Reply:
good call. adjustment made.